Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabSaveLoadHandler.h

```diff
@@ -60,6 +60,7 @@ namespace AzToolsFramework
 
             // Dialogs
             int ExecuteClosePrefabDialog(TemplateId templateId);
+            void ExecuteSavePrefabDialog(const AZ::EntityId& entityId, bool useSaveAllPrefabsPreference);
```

Comment:
This doesn't need a bool parameter since we always want to use the preference. If we later find a use case where we shouldn't, we can add the bool parameter then. For now, this can be removed.